include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +258,1047 @@ class HighPressureGasTransport : public MultiTransport
+     *
+     * This function is structured such that it can be used for pure species or
+     * mixtures, with the only difference being the values that are passed to the
+     * function (pure values versus mixture values).
+     *
+     * For the definition of the mixture rules, see @see computeMixtureParameters() .
+     *
+     * @param Tr Reduced temperature [unitless]
+     * @param Pr Reduced pressure [unitless]
+     * @param FP_low Low-pressure polarity correction factor [unitless]
+     * @param FQ_low Low-pressure quantum correction factor [unitless]
+     * @param P_vap Vapor pressure [Pa]
+     * @param P_crit Critical pressure [Pa]
+     */
+    double highPressureNondimensionalViscosity(double Tr, double Pr, double FP_low,
+                                              double FQ_low, double P_vap,
+                                              double P_crit);
+
+    /**
+     * Calculates quantum correction term of the Lucas method for a species based
+     * on the reduced temperature(Tr) and molecular weight(MW), used in viscosity
+     * calculation.
+     *
+     * @f[
+     *  F_{Q}^{\text{o}} = 1.22 Q^{0.15} {1 + 0.00385[ (T_r - 12)^2 ]^{\frac{1}{MW}}
+     *                     \text{sign} (T_r - 12 )}
+     *
+     *  \quad \text{(Equation 9-4.19)}
+     * @f]
+     *
+     * @param Q  Species-specific constant
+     * @param Tr  Reduced temperature [unitless]
+     * @param MW  Molecular weight [kg/kmol]
+     */
+    double quantumCorrectionFactor(double Q, double Tr, double MW);
+
+    /**
+     * Returns the polarity correction term for a species based on reduced temperature,
+     * reduced dipole moment, and critical compressibility. Used in the calculation of
+     * viscosity.
+     *
+     * Calculates polarity correction term of the Lucas method for a species based
+     * on the reduced temperature(Tr) and molecular weight(MW). Equation 9.4.18.
+     *
+     * @f[
+     *  \begin{equation}
+     *   F_P^0 =
+     *   \begin{cases}
+     *      1 & 0 \leq \mu_r < 0.022 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} & 0.022 \leq \mu_r < 0.075 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} \times 0.96 + 0.1(T_r - 0.7) & 0.075 \leq \mu_r
+     *   \end{cases}
+     *  \end{equation}
+     * @f]
+     *
+     * @note The original description in Poling(2001) neglects to mention what happens
+     * when the quantity raised to the 1.72 power goes negative. That is an undefined
+     * operation that generates real and imaginary numbers. For now, only positive
+     * values are allowed.
+     *
+     * @param mu_r  Species Reduced dipole moment
+     * @param Tr  Reduced temperature
+     * @param Z_c  Species Critical compressibility
+     */
+    double polarityCorrectionFactor(double mu_r, double Tr, double Z_c);
+
+
+    private:
+    //! Critical properties of each species.
+    std::vector<double> m_Tcrit;
+    std::vector<double> m_Pcrit;
+    std::vector<double> m_Vcrit;
+    std::vector<double> m_Zcrit;
```

Comment:
```suggestion

private:
    //! Critical properties of each species.
    vector<double> m_Tcrit;
    vector<double> m_Pcrit;
    vector<double> m_Vcrit;
    vector<double> m_Zcrit;
```
Also, as far as Doxygen is concerned, the comment only applies to `m_Tcrit`.

Same applies to all other use of `std::vector`, which does not need to be prefixed due to a `using std::vector` statement in `ct_defs.h`.